persistence-cassandra-4.0/src/main/java/io/stargate/db/cassandra/impl/Conversion.java

```diff
@@ -438,8 +438,18 @@ private static PersistenceException addSuppressed(
 
   public static Result toResult(
       ResultMessage resultMessage, org.apache.cassandra.transport.ProtocolVersion version) {
-    return toResultInternal(resultMessage, version)
-        .setTracingId(ReflectionUtils.getTracingId(resultMessage));
+    return toResult(resultMessage, version, true);
+  }
+
+  public static Result toResult(
```

Comment:
Should we create an issue to add this to the other persistence modules?

Reply:
Not sure if it's needed.. Here it's really more about avoid that reflection call if you know trace information will not be needed..